emmarin/cl/ledger/src/ledger.rs

```diff
@@ -19,14 +19,14 @@ pub struct ProvedLedgerTransition {
 
 // TODO: find a better name
 #[derive(Debug, Clone)]
-pub struct ProvedZoneTx {
-    pub bundle: ProvedBundle,
+pub struct ProvedBundle {
+    pub bundle: ProvedBalance,
```

Comment:
rename to balance?

Reply:
not sure what this comment means. I've changed the name from bundle proof to balance proof, because that was all it ws doing. This here is instead a bundle because it contain the full information about ptxs included and balance proof

Reply:
I was suggesting to rename the `bundle` field here to `balance` since it's a proved balance